content/en/guide/v10/getting-started.md

```diff
@@ -68,23 +68,23 @@ For more information on HTM, check out its [documentation][htm].
 
 [Preact CLI] is an off-the-shelf solution for building Preact applications that is optimized for modern web development. It's built on standard tooling projects like Webpack, Babel and PostCSS. Preact CLI does not require any configuration or prior knowledge to get started, and this simplicity makes it the most popular way to use Preact.
 
-As the name implies, Preact CLI is a **c**ommand-**li**ne tool that can be run in the terminal on your machine. Install it globally by running:
+As the name implies, Preact CLI is a **c**ommand-**li**ne tool that can be run in the terminal on your machine. It can be added to your current project by running:
 
 ```bash
-npm install -g preact-cli
+npm install --save-dev preact-cli
 ```
 
-After that, you'll have a new command in your terminal called `preact`. Using it, you can create a new application by running the `preact create` command:
+After that, you'll have a new command in your project called `preact`. Using it, you can create a new application by running the `preact create` command:
 
```

Comment:
I'd say this whole block could go really. I don't think people will add CLI to an existing project often, and the non-global binary usage I think will screw up project creation.

Reply:
Good point. PR for that: https://github.com/preactjs/preact-www/pull/778#discussion_r653665266